tests/e2e_tests/utils.py

```diff
@@ -1,17 +1,19 @@
 import os
+import requests
 import shutil
 import subprocess
 import sys
 import time
-from typing import List
+from typing import List, Optional
 
 from substrateinterface import SubstrateInterface
 
 import bittensor
 from bittensor import Keypair, logging
 
 template_path = os.getcwd() + "/neurons/"
-repo_name = "templates repository"
+templates_repo = "templates repository"
+ASYNC_TEMPL_URL = "https://api.github.com/repos/opentensor/bittensor-subnet-template/commits/async-metagraph-for-async-e2e-tests-only"
```

Comment:
I don't like this being a hardcoded github link for longterm maintenance. Add a TODO comment here.

Reply:
There is an expression: if you deny something, then offer a solution ;)
Now this is used not just as hardcode, but as a constant within the test. Our goal is to provide the opportunity for this constant to exist outside the test. No matter how we try to do this, inside the test it will all be a constant.
Does it make sense?